hicexplorer/hicPlotMatrix.py

```diff
@@ -424,14 +424,16 @@ def getRegion(args, ma):
     chrom = region_start = region_end = idx1 = start_pos1 = chrom2 = region_start2 = region_end2 = idx2 = start_pos2 = None
     chrom, region_start, region_end = translate_region(args.region)
 
-    # print("len(cut_intervals)", len(ma.cut_intervals))
-    if type(next(iter(ma.interval_trees))) is np.bytes_:
+    if type(next(iter(ma.interval_trees))) is np.bytes_ or type(next(iter(ma.interval_trees))) is bytes:
```

Comment:
`if type(next(iter(ma.interval_trees))) in [np.bytes_, bytes]`

Reply:
Ah, I was searching for something like this. :) Thanks.